articles/building-a-simple-cryptocurrency-blockchain/index.md

```diff
@@ -0,0 +1,175 @@
+## Building Simple cryptocurrency system using Node.js
+
+In this tutorial, we are going to demonstrate how to build a simple cryptocurrency system, called thecoin.
```

Comment:
Could you please write a more brief intro about what the goals of this article are? the key takeaways? what are you going to demonstrate? And, explain about "thecoin" in detail

Reply:
Talk about what blockchain is, then talk about what cryptocurrencies are

Reply:
Done

Reply:
Done